### ISSUE TITLE
Running "assets:install" results in an empty "bundles" directory

### ISSUE THREAD

**Issue by superhaggis** [symfony/symfony#23405]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | yes
| RFC?             | no
| Symfony version  | 3.2.10

After upgrading Symfony from 3.2.9 to 3.2.10, running the "assets:install" command results in an empty "bundles" directory within the "web" directory - none of the individual bundle directories and assets are copied across.  The command output reported that the files were copied.

Downgrading to 3.2.9 and running "assets:install" again resulted in expected behaviour.

**Comment by fabpot** [comment]
Already fixed.

**Comment by xabbuh** [comment]
@superhaggis see #23378

**Comment by superhaggis** [comment]
@xabbuh What I'm saying is that it worked in 3.2.9 but is now broken in 3.2.10.  We never had an issue until we upgraded.

**Comment by superhaggis** [comment]
```
$ php -v
PHP 7.0.17 (cli) (built: Mar 14 2017 16:34:11) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.17, Copyright (c) 1999-2017, by Zend Technologies
    with Xdebug v2.5.1, Copyright (c) 2002-2017, by Derick Rethans

$ php bin/console -V
Symfony 3.2.9 (kernel: app, env: dev, debug: true)

$ php bin/console assets:install
 Installing assets as hard copies.

 ---- ------------------------------ ----------------
       Bundle                         Method / Error
 ---- ------------------------------ ----------------
  OK   NelmioApiDocBundle             copy
  OK   FOSJsRoutingBundle             copy
  OK   WhiteOctoberPagerfantaBundle   copy
  OK   StarRatingBundle               copy
  OK   AppBundle                      copy
 ---- ------------------------------ ----------------

 ! [NOTE] Some assets were installed via copy. If you make changes to these assets you have to run this command again.


 [OK] All assets were successfully installed.

$ ls -al web/bundles/
drwxrwx---+ 1 user group 0 Jul  5 11:22 .
drwxrwx---+ 1 user group 0 Jul  5 11:21 ..
drwxrwx---+ 1 user group 0 Jul  5 11:22 app
drwxrwx---+ 1 user group 0 Jul  5 11:22 fosjsrouting
drwxrwx---+ 1 user group 0 Jul  5 11:22 nelmioapidoc
drwxrwx---+ 1 user group 0 Jul  5 11:22 starrating
drwxrwx---+ 1 user group 0 Jul  5 11:22 whiteoctoberpagerfanta

$ nano composer.json <locked symfony/symfony to 3.2.10>
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating symfony/symfony (v3.2.9 => v3.2.10): Loading from cache
...

$ php bin/console -V
Symfony 3.2.10 (kernel: app, env: dev, debug: true)

$ php bin/console assets:install

 Installing assets as hard copies.

 ---- ------------------------------ ----------------
       Bundle                         Method / Error
 ---- ------------------------------ ----------------
  OK   NelmioApiDocBundle             copy
  OK   FOSJsRoutingBundle             copy
  OK   WhiteOctoberPagerfantaBundle   copy
  OK   StarRatingBundle               copy
  OK   AppBundle                      copy
 ---- ------------------------------ ----------------

 ! [NOTE] Some assets were installed via copy. If you make changes to these assets you have to run this command again.


 [OK] All assets were successfully installed.

$ ls -al web/bundles/
total 8
drwxrwx---+ 1 user group 0 Jul  5 11:28 .
drwxrwx---+ 1 user group Users 0 Jul  5 11:26 ..
```

The commands were executed in a Cygwin command window under Windows 10 x64.

**Comment by xabbuh** [comment]
@superhaggis That's because a commit added in 3.2.10 caused this issue which was fixed afterwards and will be part of 3.2.11.

**Comment by superhaggis** [comment]
@xabbuh Okay, thanks. 😃
